apps/minaintyg/src/mocks/certificateContentMock.ts

```diff
@@ -0,0 +1,148 @@
+export const certificateContentMock = [
```

Comment:
We should have a talk within the team if the model is changing. The backend solution is built according to the original model we designed.